mass_mailing_list_dynamic/models/mail_mass_mailing_list.py

```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models
-from odoo.tools import safe_eval
+from odoo.tools.safe_eval import safe_eval
```

Comment:
Why changing this? It was working before.

Reply:
correct - was working before - i thought it would be better to directly only import the function... - i will undo this